packages/css/src/types.ts

```diff
@@ -14,7 +9,7 @@ type StandardCSSProperties = CSS.Properties<number | string>
  *
  * For more information see: https://styled-system.com/responsive-styles
  */
-export type ResponsiveStyleValue<T> = T | Array<T | null>
+export type ResponsiveStyleValue<T> = T | Array<T | null | undefined>
```

Comment:
I needed it to make _@theme-ui/css_ strict TS, but it makes sense in general. `array[index]` can always be undefined.